metanode/partition.go

```diff
@@ -1178,9 +1178,20 @@ func (mp *metaPartition) store(sm *storeMsg) (err error) {
 	}
 
 	// write crc to file
-	if err = os.WriteFile(path.Join(tmpDir, SnapshotSign), crcBuffer.Bytes(), 0o775); err != nil {
-		return
+	crcFile, err := os.OpenFile(path.Join(tmpDir, SnapshotSign), os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0o775)
```

Comment:
It's better to add  `util/fileutil.WriteFileSafe` for implementing sync writing file.

Reply:
Done. Looks good.